src/main/webapp/app/admin/admin.route.ts

```diff
@@ -1,19 +1,36 @@
+<<<<<<< HEAD
+import { Routes } from '@angular/router';
+
+import {
+    configurationRoute,
+    healthRoute,
+    logsRoute,
+    metricsRoute
+=======
 import { Routes, CanActivate } from '@angular/router';
 
 import {
     // configurationRoute,
     healthRoute,
     // logsRoute,
     // metricsRoute
+>>>>>>> refs/remotes/origin/angular-2
```

Comment:
You need to remove the useless part of this change
+<<<<<<< HEAD
...
+=======
....
+>>>>>>> refs/remotes/origin/angular-2